test/javascript/turbolinks_visit_test.coffee

```diff
@@ -315,3 +315,16 @@ suite 'Turbolinks.visit()', ->
       assert.equal @window.pageYOffset, 42
       done()
     @Turbolinks.visit('iframe2.html', scroll: false)
+
+  test "with :title set to a value replaces the title with the value", (done) ->
```

Comment:
Also could you move these tests before [this line](https://github.com/gmalette/turbolinks/blob/8256f7ede80458d9f8be389c87c9f1b389899dd9/test/javascript/turbolinks_visit_test.coffee#L235) so they run on CI.